team/template.yaml

```diff
@@ -0,0 +1,17 @@
+- title: Your name goes here
+  avatar: A URL with your picture
+  role: Your title
+  organizations:
+    - name: Your institution
+  bio: A short bio about yourself.
+    You can have multiple lines. Just watch that you have two leading spaces at the beginning.
+  expertise:
+    - skill 1
+    - skill 2
+  social:
+    - icon: github
+      icon_pack: fab
```

Comment:
Any way we can tell people to leave this as 'fab'? It's not immediately obvious to a new person.

Reply:
We could add an inline comment